### ISSUE TITLE
[Error] distrobox-based containerized application leaves behind temporary files

### ISSUE THREAD

**Issue by mcepl** [89luca89/distrobox#1623]
**Describe the bug**

I am trying to use a `distrobox`-run container as an application, specifically my `$EDITOR`. Specifically, the project is hosted at https://build.opensuse.org/package/show/home:mcepl:moldavite/vis-master-container.

Everything works fine with slightly modified `distrobox-export` generated script, but when the editor leaves, there is sometime a temporary file left behind (.*.vis.* pattern).

```sh
#!/bin/sh
# distrobox_binary
# name: vis
# outside of any container
if [ -z "${CONTAINER_ID}" ]; then
   exec /usr/bin/distrobox-enter  -n vis --  /usr/bin/vis  "$@"
# inside of the vis container
elif [ -x "/usr/bin/vis" ] ; then
    /usr/bin/vis "$@"
# inside of other container
elif [ -x "/usr/bin/distrobox-host-exec" ] ; then
    /usr/bin/distrobox-host-exec ~/.bin/vis "$@"
else
    echo "Cannot run vis"
    exit 255
fi
```

The only problem is that this editor leaves behind zero-length temporary files named `.*.vis.*` (e.g., `~/projekty/git-fixup/.git/.COMMIT_EDITMSG.vis.Hf71uh`). I think those files I created at https://github.com/martanne/vis/blob/master/text-io.c#L253, and I wonder how it could clash with distrobox.

This change https://github.com/martanne/vis/pull/507 looks a bit suspicious to me, but I wonder whether it is truly an error and whether distrobox shouldn’t accommodate it.

The problem doesn’t happen 100% (perhaps `vis` doesn’t create temporary files with small files?), so [this log of `distrobox-enter`](https://github.com/user-attachments/files/17959754/distrobox-enter-log.txt) doesn’t show anything which I would consider wrong.

**Desktop (please complete the following information):**
 - Are you using podman, docker or lilipod? podman
 - Which version or podman, docker or lilipod? 5.2.5
 - Which version of distrobox? 1.8.0
 - Which host distribution? MicroOS (packages from openSUSE/Tumbleweed)
 - How did you install distrobox? system packages

**Comment by mcepl** [comment]
[vis-debug-log.txt](https://github.com/user-attachments/files/18058629/vis-debug-log.txt)

This is the output of one session where this creation of hidden files happened, but I don’t see anything interesting there.

**Comment by 89luca89** [comment]
If these files are created by the application I don't see how I can prevent this 🤷

**Comment by mcepl** [comment]
I was not asking so much for you doing something, but for suggesting if you have any idea what’s going on. However, only now I see that this project actually has discussions switched on. I will move this question there, then. I am sorry for bothering you here.

**Comment by mcepl** [comment]
Right, I have actually filed that https://github.com/89luca89/distrobox/discussions/1398 but it was ignored. Oh well.
